docs/models/policyrule.md

```diff
@@ -22,7 +22,12 @@ It is recommended to use a descriptive name to best identify a PolicyRule, other
 * Action (string, choice of deny drop allow)
 * Log (boolean)
 * Request ID (optional, string)
-    * Meant to represent an upstream request (e.g. an service request from an ITSM solution).
+  * Meant to represent an upstream request (e.g. an service request from an ITSM solution).
```

Comment:
the new rtd uses 4 spaces for these bullets

Reply:
Will revert, markdown plugin for vs code was yelling